smart_contract/executor/src/permission.rs

```
@@ -53,6 +53,8 @@ pub enum PermissionTokenConversionError {
     Id(PermissionTokenId),
     /// Failed to deserialize JSON
     Deserialize(serde_json::Error),
+    /// Failed to serialize JSON
+    Serialize(serde_json::Error),
```

this is unnecessary, we should panic on invalid Executor